### ISSUE TITLE
nan encountered

### ISSUE THREAD

**Issue by Vinnni002** [mRcfps/WESUP#2]
while training just after 1st iteration in 1st epoch, loss value becomes nan. How to solve this?

**Comment by AhmadObeid** [comment]
Indeed, right after it starts training following all the instructions it encounters a nan value and raises an error and stops the trianing.
Any solutions?
